model/src/main/kotlin/OrtResult.kt

```diff
@@ -237,6 +237,25 @@ data class OrtResult(
         return vendorPackages
     }
 
+    /**
+     * Returns the path of the folder that contains the definition file of the [project], relative to the root VCS.
+     */
+    fun getCompleteDefinitionFilePath(project: Project): String {
+        val vcsPath = repository.getVcsPath(project.vcsProcessed)
+
+        requireNotNull(vcsPath) {
+            "The VCS ${project.vcs} is not contained in the repositories of this ORT result: $repository"
+        }
+
+        return buildString {
+            if (vcsPath.isNotBlank()) {
```

Comment:
`isNotEmpty()` should be enough here as we have full control here about the created `vcsPath`.

Reply:
Done.